plugins/woocommerce/client/admin/client/settings-payments/components/payment-gateway-buttons/payment-gateway-buttons.tsx

```diff
@@ -105,8 +109,19 @@ export const PaymentGatewayButton = ( {
 		return enabled ? textSettings : textEnable;
 	};
 
+	const activatePayments = () => {
+		setIsActivatingPayments( true );
+
+		window.location.href = getWooPaymentsSetupLiveAccountLink();
```

Comment:
I am not sure, here we need to direct merchant to our WooPayments connect page, we do need page reload, so I believe in this case router wouldn't work, but if it's SPA and we don't redirect but navigate, it will be so. BTW I am not fimiliar how React routing is done in WooCommerce yet.